Dockerfile

```diff
@@ -4,6 +4,8 @@ MAINTAINER Michael J. Stealey <stealey@renci.org>
 ENV DEBIAN_FRONTEND noninteractive
 ENV PY_SAX_PARSER=hs_core.xmlparser
 
+RUN printf "deb http://archive.debian.org/debian/ jessie main\ndeb-src http://archive.debian.org/debian/ jessie main\ndeb http://security.debian.org jessie/updates main\ndeb-src http://security.debian.org jessie/updates main" > /etc/apt/sources.list
```

Comment:
are all of these sources actually necessary?

Reply:
The build would fail without the line.

Reply:
Good catch. I am not sure about deb-src. We have something similar in the docker.list
`deb https://apt.dockerproject.org/repo debian-jessie main`

Reply:
deb-src is used to grab the source package if we want to compile. I don't think the deb-src it isn't necessary. Let me try one more  build without it.

Reply:
@sblack-usu I attempted to build a image from scratch. It appeared to take more than two hours (still in progress) on my dev-hs-5. Is this normal?

Reply:
yes, that is normal.  If you're that far it's past installing the package so it should be good.

Reply:
right, it went through.